templates/pharmacies/general_edit.html

```diff
@@ -45,7 +45,18 @@
 		<input TYPE="TEXT" NAME="fax" SIZE="12" VALUE="{$pharmacy->get_fax()}" onKeyDown="PreventIt(event)" />
 	</td>
 </tr>
-
+<tr>
+    <td VALIGN="MIDDLE" >{xl t='NPI'}</td>
+    	<td VALIGN="MIDDLE" >
+		<input TYPE="TEXT" NAME="npi" SIZE="12" VALUE="{$pharmacy->get_npi()}" onKeyDown="PreventIt(event)" />
+	</td>
+</tr>
+<tr>
+	<td VALIGN="MIDDLE" >{xl t='NCPDP'}</td>
```

Comment:
The NCPDP Processor ID Number (BIN) is a six-digit number that health plans can use to process electronic pharmacy claims if they do not use pharmacy benefit cards with a magnetic stripe.
https://www.ncpdp.org/Resources/NCPDP-Processor-ID-(BIN)